apps/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -1,35 +1,34 @@
-import { Component, Input } from '@angular/core';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 import { StudentStore } from '../../data-access/student.store';
 import { TeacherStore } from '../../data-access/teacher.store';
 import { CardType } from '../../model/card.model';
 
 @Component({
   selector: 'app-list-item',
   template: `
+    <!-- <ng-template #appListItemTemplate let-item="item"> -->
     <div class="border border-grey-300 py-1 px-2 flex justify-between">
       {{ name }}
       <button (click)="delete(id)">
         <img class="h-5" src="assets/svg/trash.svg" />
       </button>
     </div>
+    <!-- </ng-template> -->
   `,
   standalone: true,
 })
 export class ListItemComponent {
   @Input() id!: number;
   @Input() name!: string;
   @Input() type!: CardType;
+  @Output() deleteEvent = new EventEmitter<number>();
 
   constructor(
     private teacherStore: TeacherStore,
     private studentStore: StudentStore
```

Comment:
you can delete those

Reply:
hello i dont get it, if i delete it how am i going to delete ?.. That's what am using to delete an item

Reply:
for me, list item is just a container with a delete button, the item doesn't need to have any information. The parent can hold the information. So the parent will listen to the delete event and give the correct information at that time (in the parent).

Reply:
Made changes .. thanks very much!

Reply:
that's way better 👍

Reply:
thank you 👍